src/utils/metrics.js

```diff
@@ -84,10 +99,22 @@ export let modalKind = (() => {
   return shouldShowModalNewsletter ? "newsletter" : "sticky";
 })();
 
+export let modalTakeover = (() => {
+  if (window.matchMedia("(prefers-reduced-motion), (speech)").matches) {
+    // eslint-disable-next-line no-console
+    console.warn("prefers-reduced-motion; aborting modal display");
+    return "none";
+  }
+  return shouldShowModalTakeover ? "takeover" : "sticky";
+})();
```

Comment:
Instead of having a new variable for modalTakeover, I think this should incorporated into modalKind, like this:

```js
export let modalKind = (() => {
  if (window.matchMedia("(prefers-reduced-motion), (speech)").matches) {
    // eslint-disable-next-line no-console
    console.warn("prefers-reduced-motion; aborting modal display");
    return "none";
  }
  if (shouldShowModalTakeover) {
    return "takeover";
  }
  if (shouldShowModalNewsletter) {
    return "newsletter";
  }
  return "sticky";
})();
```

Reply:
Hmm, that won't work because the JS doesn't know if the takeover is turned on or off. Leave it your way for now.